materials/sql_joins.md

```diff
@@ -46,3 +46,5 @@ SELECT species, scientificname
 FROM Main
 JOIN Species WHERE species = new_code;
 ```
```

Comment:
Agreed. Also let's go ahead and drop the "No ON clause" example since that's *definitely* not worth teaching anyone.